src/modules/terminalfont.c

```diff
@@ -179,6 +179,40 @@ static void printXCFETerminal(FFinstance* instance)
     }
 }
 
+static void printAlacritty(FFinstance* instance) {
+    FFstrbuf fontName;
+    FFstrbuf fontSize;
+    ffStrbufInit(&fontName);
+    ffStrbufInit(&fontSize);
+
+    FFpropquery fontQuery[] = {
+        {"family:", &fontName},
+        {"size:", &fontSize},
+    };
+
+    // alacritty parses config files in this order
+    ffParsePropFileConfigValues(instance, "alacritty/alacritty.yml", 2, fontQuery);
+    if(fontName.length == 0 || fontSize.length == 0)
+        ffParsePropFileConfigValues(instance, "alacritty.yml", 2, fontQuery);
+    if(fontName.length == 0 || fontSize.length == 0)
+        ffParsePropFileConfigValues(instance, ".alacritty.yml", 2, fontQuery);
+    if(fontName.length == 0 || fontSize.length == 0)
+    {
+         //by default alacritty uses it's own font called alacritty at size 11
+        ffStrbufAppendS(&fontName, "alacritty");
+        ffStrbufAppendS(&fontSize, "11");
+    }
+
+    FFfont font;
+    ffFontInitCopy(&font, fontName.chars);
+    ffStrbufInitCopy(&font.size, &fontSize);
```

Comment:
`font.size` is already initialized by fontInitCopy. A simple `ffStrbufAppend` should do the trick. Otherwise we leak memory.